exchanges/binance/binance_wrapper.go

```diff
@@ -189,10 +189,12 @@ func (b *Binance) SetDefaults() {
 			},
 		},
 		Subscriptions: subscription.List{
-			{Enabled: true, Channel: subscription.TickerChannel},
```

Comment:
In binance.go `subscriptionNames` map can be removed

Reply:
Yep, thanks.
Note: This is not a universal truth; Some other exchanges need to keep a map for resp mapping.

Fixed 808dfe1